.travis.yml

```diff
@@ -1,6 +1,8 @@
 sudo: false
 language: java
 install: 
+  # install the maven wrapper
+  - mvn -N io.takari:maven:wrapper -Dmaven=3.3.9
```

Comment:
Could we have a comment explaining why we do this and what it is? I know in 6 months I won't remember.